api/src/main/java/jakarta/servlet/http/HttpServlet.java

```diff
@@ -488,6 +491,11 @@ protected void doTrace(HttpServletRequest req, HttpServletResponse resp) throws
 
         while (reqHeaderEnum.hasMoreElements()) {
             String headerName = reqHeaderEnum.nextElement();
+
+            if (isSensitiveHeader(headerName)) {
+                continue;
```

Comment:
Should the headers be skipped, or should just there values be obscured?  RFC kind of says both.

Reply:
Maybe obscure can show that theses headers exists. Should the response knows theses headers are present?

Reply:
I think leave it as skipping.  If somebody wants to see more then they can implement doTrace themselves.